tests/fast-integration/logging/index.js

```diff
@@ -22,7 +22,7 @@ function istioAccessLogsTests(startTimestamp) {
 
     it('Should create the Istio Access Logs resource for Loki', async () => {
       await k8sApply(istioAccessLogsResource, namespace);
-      await sleep(10000); // wait 10 seconds until resource is ready
+      await sleep(20000); // wait 20 seconds until resource is ready
```

Comment:
Can we find a better way to validate that the resource is ready? Does it have some status attribute?

Reply:
No, there is no status field

```bash 
$ k describe telemetries.telemetry.istio.io access-config
Name:         access-config
Namespace:    kyma-system
Labels:       <none>
Annotations:  <none>
API Version:  telemetry.istio.io/v1alpha1
Kind:         Telemetry
Metadata:
  Creation Timestamp:  2022-08-08T14:46:33Z
  Generation:          1
  Managed Fields:
    API Version:  telemetry.istio.io/v1alpha1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          .:
          f:kubectl.kubernetes.io/last-applied-configuration:
      f:spec:
        .:
        f:accessLogging:
        f:selector:
          .:
          f:matchLabels:
            .:
            f:service.istio.io/canonical-name:
    Manager:         kubectl-client-side-apply
    Operation:       Update
    Time:            2022-08-08T14:46:33Z
  Resource Version:  8220
  UID:               432600b4-d243-4670-88c0-c400d583e877
Spec:
  Access Logging:
    Providers:
      Name:  stdout-json
  Selector:
    Match Labels:
      service.istio.io/canonical-name:  loki
Events:                                 <none>
```